### PR TITLE
Include scene trends in feeds

### PR DESCRIPTION
**Pull request by devinivy**
This adds trends into the author feed and timeline.  Those schemas were modified by adding `trendedBy` (a la `repostedBy`) to the feed item and `actorType` to all actor references within the feed items.

### CODE REVIEW

**Comment by pfrazee** [comment]
lgtm

File: packages/pds/tests/seeds/client.ts [hunk 1]
```diff
@@ -219,12 +233,14 @@ export class SeedClient {
         encoding: 'application/json',
       },
     )
+    this.dids[handle] = res.data.did
```
Comment (devinivy) [inline packages/pds/tests/seeds/client.ts:236]:
```suggestion
```

File: packages/pds/tests/views/notifications.test.ts [hunk 2]
```diff
@@ -70,13 +70,16 @@ describe('pds notification views', () => {
   })
 
   it('paginates', async () => {
+    // @TODO there is a small bug where notifications generated with the same
```
Comment (dholms) [inline packages/pds/tests/views/notifications.test.ts:73]: yeah :/ this is actually an issue with all our pagination i believe :/

**Review by dholms — APPROVED** [review]
awesome thanks for tackling this 🙌

File: packages/pds/tests/views/notifications.test.ts (hunk 2 above)
Comment (devinivy) [reply packages/pds/tests/views/notifications.test.ts:73]: I think you're right.  The exception is the actor search, whose cursor is a pair `(score, did)`.

**Pull request merged**